hscontrol/templates/apple.go

```diff
@@ -0,0 +1,149 @@
+package templates
+
+import (
+	"fmt"
+
+	"github.com/chasefleming/elem-go"
+	"github.com/chasefleming/elem-go/attrs"
+)
+
+func Apple(url string) *elem.Element {
+	return HtmlStructure(
+		elem.Title(nil,
+			elem.Text("headscale - Apple")),
+		elem.Body(attrs.Props{
+			attrs.Style: bodyStyle.ToInline(),
+		},
+			headerOne("headscale: iOS configuration"),
+			headerTwo("GUI"),
+			elem.Ol(nil,
+				elem.Li(nil,
+					elem.Text("Install the official Tailscale iOS client from the"),
+					elem.A(attrs.Props{attrs.Href: "https://apps.apple.com/app/tailscale/id1470499037"},
+						elem.Text("App store"),
+					),
+				),
+				elem.Li(nil,
+					elem.Text("Open Tailscale and make sure you are"),
+					elem.I(nil, elem.Text("not")),
```

Comment:
When rendered, there is no space between those elements:


![image](https://github.com/user-attachments/assets/8f2c9a4c-65ad-47d1-9d69-b3ab0dd9a804)